sdk/src/test/java/com/commutestream/sdk/AgencyInterestTest.java

```diff
@@ -17,12 +17,12 @@ public void equals() {
         AgencyInterest aiAgencyVaries = new AgencyInterest(AgencyInterest.TRACKING_DISPLAYED, "paac", "Red", "12345");
```

Comment:
Perhaps this doesn't really matter, but for the record "paac" should be "PAAC". "cta" is our only lowercase agency ID.